hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java

```diff
@@ -275,7 +275,6 @@ private void validateRoyaltyFeeForCreation(
                 validateTrue(denominatingTokenId.tokenNum() != 0, CUSTOM_FEE_DENOMINATION_MUST_BE_FUNGIBLE_COMMON);
                 validateExplicitTokenDenomination(
                         fee.feeCollectorAccountId(), denominatingTokenId, tokenRelationStore, tokenStore);
-                fees.add(fee);
```

Comment:
What's the significance of this deletion?

Reply:
FeeCollector specified in `fallBackFee` should not be automatically associated as I see it. So, deleted from adding to requireAutomaticAssociations

Reply:
Ohhh...got it, that makes sense.